src/web/tags.rs

```diff
@@ -65,14 +66,14 @@ impl Tags {
                 insert_if_not_empty("ua.name", ua_result.name, &mut tags);
                 insert_if_not_empty("ua.os.ver", &ua_result.os_version.to_owned(), &mut tags);
                 insert_if_not_empty("ua.browser.ver", ua_result.version, &mut tags);
+                extra.insert("ua".to_owned(), uas.to_string());
             }
         }
-        // `uri.path` causes too much cardinality for influx.
         tags.insert("uri.method".to_owned(), req_head.method.to_string());
-        Tags {
-            tags,
-            extra: HashMap::new(),
-        }
+        // `uri.path` causes too much cardinality for influx but keep it in
+        // extra for sentry
```

Comment:
Kinda wonder if we should add a comment to the `Tags` struct to note that `extra` is ONLY used by sentry. That or change it to be something like `sentry_tags`